components/Text/text.css

```diff
@@ -0,0 +1,65 @@
+@import url('/utils.css');
+.heading-container{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.h1{
+    font-size: 3.5rem;
+}
+.h2{
+    font-size: 3rem;
+}
+.h3{
+    font-size: 2.5rem;
+}
+.h4{
+    font-size: 2rem;    
+}
+.h5{
+    font-size: 1.75rem;
+}
+.h6{
+    font-size: 1.25rem;
+}
+
+.xs-txt{
```

Comment:
for grouping, it would be good if you name it as txt-$$ instead of $$-txt and that's the general naming convention followed by all libraries

Reply:
okay I'll follow that

Reply:
I have made the changes